tabu/sampler.py

```diff
@@ -228,7 +235,8 @@ def _bqm_to_tabu_qubo(bqm):
         ud *= .5
         symm = ud + ud.T
         qubo = symm.tolist()
-        return qubo, varorder
+        maxbias = max(symm.max(), -symm.min())
```

Comment:
It might make sense to check that case that they are all naturally integers. Something like `(symm == symm.floor()).all()`.

Reply:
Why? It's ok to get a float for maxbias

Reply:
Sorry, I annotated the wrong line. We won't need to scale in the case that they are all already integer.

Reply:
Yes, that's a good idea.